.github/file-paths.yaml

```diff
@@ -26,7 +26,7 @@ frontend_sources: &frontend_sources
 frontend_specs: &frontend_specs
   - *shared_specs
   - "frontend/test/!(__support__|__runner__)/**"
-  - "frontend/**.unit.*"
+  - "frontend/**/*.unit.*"
```

Comment:
Click to see tests results:

1. [Before](https://www.digitalocean.com/community/tools/glob?comments=true&glob=frontend%2F%2A%2A.unit.%2A&matches=false&options=dot%3Atrue&tests=frontend%2Ftest%2Fmetabase%2Flib%2Ftime.unit.spec.js&tests=frontend%2Ftest%2Fmetabase%2Flib%2Fexpressions%2Ftypeinferencer.unit.spec.js&tests=frontend%2Fsrc%2Fmetabase%2Fquery_builder%2Fcomponents%2FDataSelector%2Fsaved-question-picker%2FSavedQuestionPicker.unit.spec.jsx&tests=frontend%2Fsrc%2Fmetabase%2Fcore%2Fcomponents%2FRadio%2FRadio.unit.spec.tsx)
2. [After](https://www.digitalocean.com/community/tools/glob?comments=true&glob=frontend%2F%2A%2A%2F%2A.unit.%2A&matches=false&options=dot%3Atrue&tests=frontend%2Ftest%2Fmetabase%2Flib%2Ftime.unit.spec.js&tests=frontend%2Ftest%2Fmetabase%2Flib%2Fexpressions%2Ftypeinferencer.unit.spec.js&tests=frontend%2Fsrc%2Fmetabase%2Fquery_builder%2Fcomponents%2FDataSelector%2Fsaved-question-picker%2FSavedQuestionPicker.unit.spec.jsx&tests=frontend%2Fsrc%2Fmetabase%2Fcore%2Fcomponents%2FRadio%2FRadio.unit.spec.tsx)